grimoirelab_metrics/cli.py

```diff
@@ -80,6 +80,17 @@
 @click.option("--verbose", is_flag=True, default=False, help="Increase output verbosity")
 @click.option("--code-file-pattern", help="Regular expression to match code file types")
 @click.option("--binary-file-pattern", help="Regular expression to match binary file types")
+@click.option("--pony-threshold", type=float, show_default=True, help="Pony factor threshold, between 0 and 1", default=0.5)
+@click.option(
+    "--elephant-threshold", type=float, show_default=True, help="Elephant factor threshold, between 0 and 1", default=0.5
+)
+@click.option(
+    "--dev-categories-thresholds",
+    type=str,
+    show_default=True,
+    help="Developer categories thresholds, between 0 and 1",
+    default="0.8,0.95",
+)
```

Comment:
Click has options to check that the values are between 0 and 1. I think it's something the script has to check before doing anything else.